MyConfig.h

```diff
@@ -1548,29 +1548,64 @@
 //#define MY_MQTT_SUBSCRIBE_TOPIC_PREFIX "mygateway1-in"
 
 /**
- * @def MY_MQTT_CA_CERT
- * @brief Set a specific CA certificate needed to validate MQTT server against. Use the certificate as a trust anchor, accepting remote certificates signed by it.
+ * @def MY_SSL_CERT_AUTHx
+ * @brief Up to three root Certificates Authorities could be defined to validate the mqtt server' certificate. The most secure.
+ *
+ * This define is mandatory when you need connect MQTT over SSL/TLS. Certificate Authorities.
+ * The best method to validate server certificates. 
+ * Advised to retrieve root Certificate Authorities as they expire less often than server certificates.
+ * With let's encrypt you may need up to three Certificate Authorities
  *
- * This define is mandatory when you need connect MQTT over SSL/TLS.
  * Example: @code
  *
- * const char mqtt_ca_cert[] PROGMEM = R"EOF(
+ * const char cert_isrgrootx1_Authority[] PROGMEM = R"EOF(
+ * ----- BEGIN THE CERTIFICATE -----
+ * XXX ... XXX
+ * ----- FINISH CERTIFICATE -----
+ * )EOF";
+ * 
+ * const char cert_isrgrootx2_Authority[] PROGMEM = R"EOF(
  * ----- BEGIN THE CERTIFICATE -----
  * XXX ... XXX
  * ----- FINISH CERTIFICATE -----
  * )EOF";
  *
- * #define MY_MQTT_CA_CERT mqtt_ca_cert
+ * const char cert_letsEncryptR3_Authority[] PROGMEM = R"EOF(
+ * ----- BEGIN THE CERTIFICATE -----
+ * XXX ... XXX
+ * ----- FINISH CERTIFICATE -----
+ * )EOF";
  *
+ * #define MY_SSL_CERT_AUTH1 cert_isrgrootx1_Authority
+ * #define MY_SSL_CERT_AUTH2 cert_isrgrootx2_Authority
+ * #define MY_SSL_CERT_AUTH3 cert_letsEncryptR3_Authority
+ *
+ * @endcode
+ */
+//#define MY_SSL_CERT_AUTH1
```

Comment:
In fact, I started implementing on 2.3.2 without checking if an implementation was done in dev branch. What a surprise to me when I wanted to create a PR seeing it was implemented in dev since months... Well, hopefully my contribution goes a step forward, and still thinking it's valuable. 

I'll have a look what I can do to stick with the naming and keep a compatibility layer between the both proposal.